mlflow/store/dbfs_artifact_repo.py

```diff
@@ -82,11 +87,11 @@ def log_artifacts(self, local_dir, artifact_path=None):
         for (dirpath, _, filenames) in os.walk(local_dir):
             dir_http_endpoint = root_http_endpoint
             if dirpath != local_dir:
-                rel_path = get_relative_path(local_dir, dirpath)
-                dir_http_endpoint = build_path(root_http_endpoint, rel_path)
+                rel_path = self.get_path_module().relpath(dirpath, local_dir)
```

Comment:
This is not equivalent to calling `get_relative_path`, do we know if this is a safe change?